.github/workflows/test.yml

```diff
@@ -32,17 +28,28 @@ jobs:
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - uses: conda-incubator/setup-miniconda@v2
+        with:
+          auto-update-conda: true
+          python-version: ${{ matrix.python-version }}
       - name: Install dependencies
         run: |
-          pip install torch==1.7 -f https://download.pytorch.org/whl/torch_stable.html;
+          conda install -c conda-forge rdkit
+          pip install torch==1.7.1+cpu -f https://download.pytorch.org/whl/torch_stable.html
+          pip install torchvision==0.8.2+cpu -f https://download.pytorch.org/whl/torch_stable.html
+          pip install torch-scatter -f https://pytorch-geometric.com/whl/torch-1.7.0+cpu.html
+          pip install torch-sparse -f https://pytorch-geometric.com/whl/torch-1.7.0+cpu.html
+          pip install torch-cluster -f https://pytorch-geometric.com/whl/torch-1.7.0+cpu.html
+          pip install torch-spline-conv -f https://pytorch-geometric.com/whl/torch-1.7.0+cpu.html
+          pip install torch-geometric
           pip install -e .[dev]
-        shell: bash
+        shell: bash -l {0}
       - name: Run unit tests
         run: |
           pytest --cov=kale
-        shell: bash
+        shell: bash -l {0}
       - name: Report coverage
         run: |
           coverage xml
           bash <(curl -s https://codecov.io/bash)
-        shell: bash
+        shell: bash -l {0}
```

Comment:
@pz-white May I know why you changed `bash` to `bash -l {0}`? Now [codecov](https://codecov.io/gh/pykale/pykale/tree/master/kale) is not updating and I am suspecting this to be the cause.